lib/castle-rb/api.rb

```diff
@@ -26,11 +26,12 @@ def initialize(cookie_id, ip, headers)
       @headers.delete_if { |k, v| v.nil? }
     end
 
-    def request(endpoint, args)
-      req = Net::HTTP::Post.new(
+    def request(endpoint, args, method: 'Post')
```

Comment:
Style thing: I think we should use symbols to indicate which method, ie. `def request(endpoint, args, method: :post)`, and then do String conversion when doing `const_get`. Looks pretties when using the method